### DISCUSSION TITLE
Did not install fleet during initial setup can I still install it without rebuilding everything

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#7075] [General] [answered]
Hello,
AS the subject line reads, I chose not to install OSQuery /Fleet during the initial server install for SO. 
My intent was to use Velociraptor, but that will not install properly on 2.3 so I want to install Fleet. 

How can I install Fleet, without a complete server rebuild?

**Comment by defensivedepth** [comment, marked as answer]
Unfortunately not at this time.
